presto-main/pom.xml

```diff
@@ -466,7 +465,6 @@
         <dependency>
             <groupId>com.facebook.presto</groupId>
             <artifactId>presto-function-namespace-managers</artifactId>
-            <scope>test</scope>
```

Comment:
nit: lets move this above line 384 so it's out of the testing section

Reply:
Moved